src/assets/i18n/fr.json5

```diff
@@ -697,12 +735,11 @@
   // "administrativeView.search.results.head": "Administrative Search",
   "administrativeView.search.results.head": "Recherche Administrateur",
 
-
   // "admin.workflow.breadcrumbs": "Administer Workflow",
-  "admin.workflow.breadcrumbs": "Workflow Administrateur",
+  "admin.workflow.breadcrumbs": "Gestion du Workflow",
 
   // "admin.workflow.title": "Administer Workflow",
-  "admin.workflow.title": "Workflow Adminisrateur",
+  "admin.workflow.title": "Gestion du Workflow",
```

Comment:
@mhvezina I think this is a tag in the admin menu. It leads to a page that list submissions and allow to either delete or send back to the submitter. In other word action that remove a submission from DSpace workflow. I would suggest maybe: "Dépôts en cours de traitement" or "Gestion des dépôts en cours de traitement" but its looks too long.

Reply:
@pilasou  : Bonjour Pierre. Yes.... you are right that "Gestion du Workflow" is not quite exact here.
About the use of "Workflow" : since this English term is somehow used widely, I decided to keep it instead of a more translated term like "flux de travail" or "flux des travaux". Throughout the translation work, I tried to keep "dépôt" for "repository" and "soumission" for "deposit/submission". Hence maybe we could, for now at least, go for something like "Gestion des soumisions en Workflow", what do you think?

Reply:
@mhvezina Ok, Let's try "Gestion des soumissions en Workflow" it looks long for a Menu item but it may fit.